src/project/internal/projectfilescontroller.cpp

```diff
@@ -285,16 +290,16 @@ IInteractive::Button ProjectFilesController::askAboutSavingScore(const io::path&
     return result.standardButton();
 }
 
-void ProjectFilesController::saveProject(const io::path& path)
+bool ProjectFilesController::saveProject(const io::path& path)
 {
     if (!currentNotationProject()) {
         LOGW() << "no current project";
-        return;
+        return false;
     }
 
     if (!currentNotationProject()->created().val) {
         doSaveScore();
-        return;
+        return false;
```

Comment:
This return looks confusing. Probably you need to take into account the result of doSaveScore()

Reply:
Ok, if I understand correctly, I should make doSaveScore() a bool function and return the value returned by doSaveScore()

Reply:
Yes, absolutely correct!

Reply:
I updates the pull request